spec/requests/api/v1/content_spec.rb

```diff
@@ -0,0 +1,126 @@
+RSpec.describe '/api/v1/content' do
+  include ItemSetupHelpers
+  before do
+    create :user
+  end
+
+  let(:primary_org_id) { '6667cce2-e809-4e21-ae09-cb0bdc1ddda3' }
+  let(:another_org_id) { '05e9c04d-534b-4ff0-ae8f-35c1bf9e510f' }
+
+  context 'when successful' do
```

Comment:
It might be good to have simpler test as a feature level, and move some of these
into the unit test for the ```Reports::Content```

I would only test here:

1. Happy path for 1 daily metric and 1 edition metric
2. API request errors